src/components/Comments.jsx

```diff
@@ -17,6 +17,16 @@ export default function Comments({ article_id }) {
       });
   }, [article_id]);
 
+  const handleClick = (id) => {
+    deleteCommentbyId(id).then(() => {
+      setComments((currComments)=>currComments.filter((item)=>item.comment_id!==id))
+    }) 
+      .catch((err) => {
+      console.log(err)
+    })
```

Comment:
This catch is in the right place, but instead of just a log it would be good to notify the user somehow that the delete was not successful. You could use some kind of error state, that if true displays a message for example.